ltj/urls.py

```diff
@@ -1,30 +1,15 @@
-"""ltj URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/1.9/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.conf.urls import url, include
-    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
-"""
-from django.conf.urls import include, url
+from django.urls import include, path
 from django.contrib import admin
 from django.utils.translation import ugettext_lazy as _
 
 from nature.api import router
 
 urlpatterns = [
-    url(r"^admin/", admin.site.urls),
-    url(r"v1/", include(router.urls)),
-    url(r"^ltj/", include("nature.urls")),
+    path("helauth/", include("helusers.urls")),
+    path("admin/", admin.site.urls),
+    path("v1/", include(router.urls)),
+    path("ltj/", include("nature.urls")),
 ]
 
 admin.site.site_title = _("Luontotietojärjestelmä site admin")
-admin.site.site_header = _("Luontotietojärjestelmä administration")
```

Comment:
Why was this line removed?
Not questioning the removal, probably a good point behind it, just wondering since this commits does not mention it.

Reply:
The reason for that is because `django-helusers` override the AdminSite class and make this a read-only property, which prevents us assigning a new header to it. I think I'll make a PR to the `django-helusers` project to make it possible to override the property.